core/consensus/rhd/src/lib.rs

```diff
@@ -516,14 +516,14 @@ impl<B, P, I> BftService<B, P, I>
 				start_round: 0,
 			})),
 			round_timeout_multiplier: 10,
-			key: key, // TODO: key changing over time.
+			key: key, // FIXME: key changing over time.
```

Comment:
```suggestion
			key: key,
```

removed. RHD is currently deactivated and needs to be refactored anyways.